ext/travisci/test.sh

```diff
@@ -1,22 +1,25 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 set -ueo pipefail
 set -x
 
 ulimit -u 4096
 
+run-unit-tests()
+(
+  pgdir="$(pwd)/test-resources/var/pg"
+  readonly pgdir
+
+  export PGHOST=127.0.0.1
+  export PGPORT=5432
+  test-resources/bin/setup-pdb-pg "$pgdir"
```

Comment:
Minor comment: So the test-resources directory is really for clojure related stuff, any reason why these scripts don't just live somewhere in ext/travisci or something like that? At least then all the related items are together also.